app/mailers/friends_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class FriendsMailer < ActionMailer::Base
+  layout 'mailers'
+  default from: ENV['DEFAULT_EMAIL_FROM_CARDS']
+
+  def invite(email)
+    mail(to: email, subject: 'Приглашаем Вас на сайт flashcards.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.